specs/workspace_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe "Workspace methods" do
+  describe "Select channel method" do
+    it "can return specified channel" do
+      VCR.use_cassette("select-method") do
+        test = Workspace.new
+        expect(test.select_channel("api-project")).must_be_instance_of Channel
+        expect(test.selected.name).must_equal "api-project"
+      end
+    end
+  end
+
+  describe "Select user method" do
+    it "can return a given user" do
+      VCR.use_cassette("select-user") do
+        test1 = Workspace.new
```

Comment:
What if the user you try to select doesn't exist? What happens to a previously selected user if you try to select a bad user?